config/spid-auth.php

```diff
@@ -13,6 +13,8 @@
     'sp_organization_name' => 'AGID',
     'sp_organization_display_name' => "Agenzia per l'Italia digitale",
     'sp_organization_url' => 'https://www.agid.gov.it',
+    'sp_acs_index' => env('SPID_SP_ACS_INDEX'),
```

Comment:
Credo sia necessario aggiungere un cast a `integer` oppure gestirlo in [SPIDAuth.php](https://github.com/italia/spid-laravel/blob/d9c989075f4005bee40205fe726608da10ceb716/src/SPIDAuth.php#L476-L482); il valore ritornato è infatti una `string`, di conseguenza fallisce il check nella libreria [spid-laravel](https://github.com/italia/spid-laravel).

Reply:
Effettivamente ho modificato il check in spid-laravel solo dopo aver fatto i test sul portale. :sweat_smile: